tests/unit/utils/location-test.js

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+import { describe, it, beforeEach, afterEach } from 'mocha';
+import * as LocationUtil from 'ember-simple-auth/utils/location';
+import sinonjs from 'sinon';
+
+const foo = {
+  get hash() {
+    return LocationUtil.default().hash;
+  }
+};
+
+describe('Unit | Utility | location', function() {
+  let sinon;
+  beforeEach(() => {
+    sinon = sinonjs.sandbox.create();
+  });
+  afterEach(() => {
+    sinon.restore();
+  });
+  it('works', function() {
+    expect(LocationUtil.default()).to.be.ok;
+    expect(LocationUtil.default().hash).to.be.a('string');
+  });
+  it('works when stubbed', function() {
+    expect(foo.hash).to.be.a('string');
+    sinon.stub(LocationUtil, 'default').returns({ hash: 66 });
+    expect(LocationUtil.default().hash).to.be.a('number');
+    expect(foo.hash).to.be.a('number');
+  });
```

Comment:
I'm not sure this test case is valuable as it tests the stub only which could be removed if this test case is removed?

Reply:
It was failing before I started adding sandboxes to the existing tests, and I generally like to leave tests which reveal bugs in place (to protect against regression).

Happy to remove this if you feel otherwise.

Reply:
I'd remove this test as I think it doesn't test anything that's not within this file?